### ISSUE TITLE
Not showing up in under ACP > Plugins

### ISSUE THREAD

**Issue by sbach89** [yariplus/nodebb-plugin-minecraft-integration#120]
Installed the plugin via the ACP, activated it and rebooted NodeBB, not showing up to configure.

NodeBB version is 1.10.1, fresh install.
Plugin version is 0.8.0-alpha.22

I have a feeling I'm doing something wrong, but not sure. I've even rebuilt and rebooted NodeBB.

Thanks!

**Comment by yariplus** [comment]
Taking a look!

**Comment by yariplus** [comment]
Hey. Run `git branch` for me? I think you are on the master branch. Fresh v1.10.x is coming up for me, but the master branch has a package removed that I depend on.

**Comment by sbach89** [comment]
Sorry, I was on the wrong branch haha, that's corrected but now when I click on Minecraft Integration under Plugins, it just spins.

    Loading failed for the <script> with source “https://xxxxx/assets/templates/admin/plugins/minecraft-integration.js?v=7o5cqbo6ibb”. minecraft-integration:1
    Error: Script error for "/assets/templates/admin/plugins/minecraft-integration.js"
    http://requirejs.org/docs/errors.html#scripterror require.js:7:289
    Unable to load template: admin/plugins/minecraft-integration ajaxify.js:335:3
    Error: Script error for "/assets/templates/admin/plugins/minecraft-integration.js"
    http://requirejs.org/docs/errors.html#scripterror require.js:7:289
    Unable to load template: admin/plugins/minecraft-integration ajaxify.js:335:3 


EDIT: Here are logs from nodebb related to the plugin:

    error: [Minecraft Integration] Database error getting status for server: A Minecraft Server

Haven't even set up a server yet.

**Comment by yariplus** [comment]
Looks like it might have failed in the build step. Have you done "Rebuild and Restart" from the admin page or called `./nodebb build` manually?

**Comment by sbach89** [comment]
Yep, that worked. had to rebuild and restart for another thing and now I can get in there.

Now the problem I have is the plugin and the Minecraft server aren't talking, kinda. Minecraft is SpongForge 1.12.2-2705-7.1.0-BETA-3361

When I restart nodebb, Minecraft log spews socket errors on the plugin until nodebb starts back up, so that to me looks like they are talking a little bit.

I set URL and the API key in the server, and nodebb log is saying that I need to run those commands, which I've done a bunch of times. I checked the config on the Minecraft server for the plugin, and it appears to be correct.

I downloaded the sponge plugin, should I use the universal one with SpongeForge?

**Comment by yariplus** [comment]
The sponge version should work, will verify.

**Comment by yariplus** [comment]
Ahh, it's borked. Working on fix.
